### PR TITLE
Add recipe for Ansible

### PR DESCRIPTION
**Pull request by souravsingh**

### CODE REVIEW

**Comment by souravsingh** [comment]
Can someone from@ansible help with the windows support?

**Comment by brainstorm** [comment]
@souravsingh I would suggest cutting down on the modules you are testing from ansible, some are apparently not supported on windows:

https://groups.google.com/forum/#!topic/Ansible-project/RC1bferu000

File: recipes/ansible/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+test:
+  imports:
+    - ansible
+    - ansible.constants
+    - ansible.errors
+    - ansible.utils
+    
+about:
+  home: http://github.com/ansible/ansible
+  license: GPL
```
Comment (scopatz) [inline recipes/ansible/meta.yaml]: Which version of GPL is this?

File: recipes/ansible/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+requirements:
+  build:
+    - python
+    - setuptools
+    - paramiko
+    - jinja2
+    - pyyaml
+    - httplib2
+    - pycrypto
+    - six
```
Comment (ocefpaf) [inline recipes/ansible/meta.yaml]: Usually, when using `--single-version-externally-managed`, we can remove the `run` time dependencies from the `build` section. Can you try removing the following:

```yaml
    - paramiko
    - jinja2
    - pyyaml
    - httplib2
    - pycrypto
    - six
```

and see what happens? (Sometimes the package imports itself in `setup.py` and the `run` time deps will be needed though.)

**Comment by souravsingh** [comment]
Closing this

**Comment by brainstorm** [comment]
Thanks Sourav! https://github.com/bioconda/bioconda-recipes/pull/5293 Should be closed as well, right?

**Pull request closed without merging**